RPCSX-GPU & RPCSX-OS Merge Update

Darkhost1999 opened the pull request:
Update the commands for running RPCSX
Add audio and nasm dependencies
Remove outdated information about Virtual HDD
Tease upcoming WSL information
Overall review and evaluate the information and ensure accuracy.

Darkhost1999 committed:
Update install-guide.md

1. Add Audio dependencies
2. Remove outdated Virtual HDD information no longer relevant
3. Tease WSL changes coming soon as Testers have already got it working and we just are awaiting Instructions.

misc- update commands based on building.md

Darkhost1999 committed:
Update running.md

Update changes based on recent commits merging rpcsx-gpu and rpcsx-os
All changes made are based on Usage.md

Darkhost1999 committed:
Update faq.md

Update outdated information on the FAQ

Darkhost1999 commented:
$ rubocop
Inspecting 8 files
........

8 files inspected, no offenses detected

Darkhost1999 committed:
Update install-guide.md

From today's commits add nasm

Darkhost1999 commented:
https://github.com/RPCSX/rpcsx/commit/7e7a56f0e2c96381140c149581b0bbc1a0503001

I can update running.md with arguments simplification. Or should I add the information as basic/advanced commands?

OCDkirby commented on install-guide.md:
````diff
@@ -25,51 +25,46 @@ We provide two installation methods:
+
 ```sh
-sudo pacman -S --needed git base-devel libunwind glfw-x11 vulkan-devel
+sudo pacman -S libunwind glfw-x11 vulkan-devel sox git cmake alsa-lib nasm
````
`--needed` should be preserved

OCDkirby commented:
> https://github.com/RPCSX/rpcsx/commit/7e7a56f0e2c96381140c149581b0bbc1a0503001
> 
> I can update running.md with arguments simplification. Or should I add the information as basic/advanced commands?

Maybe add it as an optional argument under see the relevant command?

Darkhost1999 committed:
Review change

TIL: --needed will install or upgrade only the packages that require updates.

Darkhost1999 commented:
I'm going to sleep. Maybe I'll add something like this in the morning.
```
##Booting Firmware
Standard Syntax
./rpcsx --system --mount /path/to/firmware/md0/ / /mini-syscore.elf

Simplified Syntax
./rpcsx --fw /path/to/firmware

##Booting Games
Standard Syntax
./rpcsx --mount /path/to/firmware/md0/ / --mount /path/to/game/app0 /app0 /app0/eboot.bin [optional args without brackets]

Simplified Syntax
./rpcsx --fw /path/to/firmware /path/to/game/

For most samples, you still need to specify mount points, e.g.
--fw /path/to/firmware--mount /path/to/api_gnm/directory/ /app0 /app0/basic-sample/basic-sample_debug.elf
```

Darkhost1999 commented:
Seems unnecessary. They can ask on Discord if syntax can be simplified but nothing wrong with being more detailed or specific about what you want to load. Especially at this time. Maybe simplify the context at 12% compatibility database for 5k games listed.
